sdktests/server_side_events_custom.go

```diff
@@ -20,64 +20,43 @@ func doServerSideCustomEventTests(t *ldtest.T) {
 	t.Run("basic properties", func(t *ldtest.T) {
 		users := NewUserFactory("doServerSideCustomEventTests-")
 		metricValue := 1.0
-
-		for _, inlineUser := range []bool{false, true} {
-			t.Run(selectString(inlineUser, "inline user", "non-inline user"), func(t *ldtest.T) {
-				for _, anonymousUser := range []bool{false, true} {
-					t.Run(selectString(anonymousUser, "anonymous user", "non-anonymous user"), func(t *ldtest.T) {
-						eventsConfig := servicedef.SDKConfigEventParams{InlineUsers: inlineUser}
-						user := users.NextUniqueUserMaybeAnonymous(anonymousUser)
-
-						dataSource := NewSDKDataSource(t, mockld.EmptyServerSDKData())
-						events := NewSDKEventSink(t)
-						client := NewSDKClient(t, WithEventsConfig(eventsConfig), dataSource, events)
-
-						client.SendCustomEvent(t, servicedef.CustomEventParams{
-							EventKey:    "event-key",
-							User:        user,
-							Data:        ldvalue.Bool(true),
-							MetricValue: &metricValue,
-						})
-
-						client.FlushEvents(t)
-						payload := events.ExpectAnalyticsEvents(t, defaultEventTimeout)
-
-						if inlineUser {
-							m.In(t).Assert(payload, m.Items(
-								m.AllOf(
-									JSONPropertyKeysCanOnlyBe("kind", "creationDate", "key", "user", "contextKind",
-										"data", "metricValue"),
-									IsCustomEvent(),
-									HasUserObjectWithKey(user.GetKey()),
-									HasContextKind(user),
-								),
-							))
-						} else {
-							m.In(t).Assert(payload, m.ItemsInAnyOrder(
-								IsIndexEvent(),
-								m.AllOf(
-									JSONPropertyKeysCanOnlyBe("kind", "creationDate", "key", "userKey", "contextKind",
-										"data", "metricValue"),
-									IsCustomEvent(),
-									HasUserKeyProperty(user.GetKey()),
-									HasContextKind(user),
-								),
-							))
-						}
-					})
-				}
+		for _, anonymousUser := range []bool{false, true} {
+			t.Run(selectString(anonymousUser, "anonymous user", "non-anonymous user"), func(t *ldtest.T) {
```

Comment:
This "anonymous vs. non-anonymous" logic will probably go away too in the new context model— its purpose in these tests was to verify that the event's `contextKind` is set properly, which will no longer be a thing.